src/overlays/actors/ovl_En_Ma1/z_en_ma1.c

```diff
@@ -402,15 +411,19 @@ void EnMa1_Update(Actor* thisx, PlayState* play) {
 
     Collider_UpdateCylinder(&this->actor, &this->collider);
     CollisionCheck_SetOC(play, &play->colChkCtx, &this->collider.base);
+
     SkelAnime_Update(&this->skelAnime);
     EnMa1_UpdateEyes(this);
+
     this->actionFunc(this, play);
+
     if (this->actionFunc != EnMa1_DoNothing) {
         Npc_UpdateTalking(play, &this->actor, &this->interactInfo.talkState, (f32)this->collider.dim.radius + 30.0f,
                           EnMa1_GetTextId, EnMa1_UpdateTalkState);
     }
-    func_80AA0B74(this);
-    func_80AA0AF4(this, play);
+
+    EnMa1_HandleSinging(this);
+    EnMa1_HandleTracking(this, play);
```

Comment:
it's more in line with the overall names to use "Update" instead of "Handle" where appropriate, I think here Update works

Reply:
+1 on this, UpdateSinging and UpdateTracking seem good, so it matches the fact that its called directly by the Update function